src/utils.ts

```diff
@@ -113,3 +113,7 @@ export function getCurrentEnv() {
   return "dev";
 }
 
+export {
+  HttpieOnHttpError,
+  RequestResponse
+};
```

Comment:
```suggestion
```
Soit il n'y a pas besoin d'export `RequestResponse`, soit il faut aussi l'exporter dans le `index.ts` auquel cas il faut juste retirer `HttpieOnHttpError`

```suggestion
export {
  RequestResponse
}
```